omegaconf/grammar/OmegaConfGrammarLexer.g4

```diff
@@ -0,0 +1,80 @@
+// Regenerate lexer and parser by running 'python setup.py antlr' at project root.
+// See `InterpolationParser.g4` for some important information regarding how to
```

Comment:
OmegaConfGrammarParser now, not InterpolationParser.

Reply:
Oops, fixed in 59e84a5